frontend/src/Layout/Header.jsx

```diff
@@ -84,18 +84,7 @@ const Header = () => {
                 </Button>
               )}
             </li>
-            <li>
-              {!auth.isLoggedIn && (
-                <Button
-                  onClick={() => {
-                    navigate("/signup");
-                  }}
-                  className="color-three"
-                >
-                  Sign Up
-                </Button>
-              )}
-            </li>
+            <li></li>
```

Comment:
Wieso ist hier jetzt noch ein leeres link item, wo vorher der Sign up Buttom war?

Reply:
Weil der Button jetzt unten in der Splashpage ist. Zwei Sign Up Buttons brauchen wir nicht.